src/components/OSCALSystemImplementation.js

```diff
@@ -73,7 +77,14 @@ export default function OSCALSystemImplementation(props) {
                     {Object.entries(props.systemImplementation.components).map(
                       ([key, component]) => (
                         <TableRow key={key}>
-                          <TableCell component="th" scope="row">
+                          <TableCell
+                            component="th"
+                            scope="row"
+                            align="left"
+                            style={{
```

Comment:
Any reason this one doesn't use a class?

Reply:
It would probably make sense to have a class for this one with the style change. Will make adjustments for this.